templates/default/event/eventinfo.html

```diff
@@ -274,11 +274,11 @@ <H1>{$layout.pagetitle}</H1>
 			{if !$event.closed}
 			    {button type="link-button" href="?m=eventedit&id={$event.id}&action=close" icon="close" label="Close"}
 			{/if}
-                        {if !$event.closed || ($event.closed && ConfigHelper::getConfig('phpui.allow_modify_closed_events_newer_than', 604800) && ($smarty.now - $event.closeddate < ConfigHelper::getConfig('phpui.allow_modify_closed_events_newer_than', 604800)))|| ConfigHelper::checkPrivilege('superuser')}
+			{if !$event.closed || ($event.closed && ConfigHelper::getConfig('phpui.allow_modify_closed_events_newer_than') && ($smarty.now - $event.closeddate < ConfigHelper::getConfig('phpui.allow_modify_closed_events_newer_than', 604800)))|| ConfigHelper::checkPrivilege('superuser')}
 			    {button type="link-button" href="?m=eventedit&id={$event.id}" icon="edit" label="Edit"}
```

Comment:
Formatowanie. To ostatnie moje przypomnienie - więcej nie będę już po prostu komentował i zostawiał PR-y otwarte bez informowania o tym ;-)

Reply:
Daj do storma jakąś konfigurację żeby było tak jak sobie życzysz.

Reply:
Kliknij Edit w webeditorze - formatowanie wygląda ok.